managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,433 @@
+      },
+      "Parameters": {
+        "VCENTER_HOST.$": "$.vcenter_host",
+        "VM.$": "$.vm"
+      }
+    },
+
+    "FailState": {
+      "Type": "Fail",
+      "Error": "FailStateError",
+      "Cause": "No Matches!"
+    },
+
+    "SuccessState": {
+      "Type": "Succeed"
+    }
+  }
+}
+```
+
+### Viewing workflow details
+
+When workflows are available, you can view the details for each workflow, such as to verify the payload. To view the details for a workflow:
+
+1. Click **Automation** > **Embedded Workflows**. Then, click **Workflows** to see the workflows list.
+
+2. Click a workflow. The default *Text* view of the workflow opens where you see the code content.
+
+   ![Text view](../images/embedworkflow_textview.png){: caption="Text view" caption-side="bottom"}
+
+   Alternatively, click the **Graph** tab to see a graph of the same data.
+
+   ![Graph view](../images/embedworkflow_graphView.png){: caption="Graph view" caption-side="bottom"}
+
+3. If your workflow has credential fields, then create a workflow credential for each of the fields. Then, map the workflow credential to the workflow.
+   <!-- Add substeps from development - A.G to provide.-->
+
+### Creating a Service Dialog using an Embedded Workflow
+
+With Embedded Workflows enabled, fields that are *Dynamic* offer the choice of *Embedded Workflows* in addition to the original *Embedded Automate*.
+
+Dialogs are the part of {{ site.data.product.title_short }} that you interface with when you order services.
+
+1. To open a service dialog, click **Automation** > **Embedded Automate** > **Customization**.
+2. From the *Service Dialog* section, review the list of created dialogs. Click one of the dialogs to view the details.
+3. Click **Edit the dialog**. You can edit the *Name*, *Provider*, and *Template*.
+
+   When you select to edit any of the dropdown settings, you can see a *dynamic* option. Toggle to **On** to make sure it is enabled.
+
+4. Click the **Options** tab. You can now select between *Embedded Automate* and *Embedded Workflows* in the Automation field.
+
+   ![Dialog Dynamic](../images/embedworkflow_servicedialogdynamic.png){: caption="Dialog Dynamic" caption-side="bottom"}
+
+   If you choose **Embedded Workflows**, you can select the workflow to run. Complete the fields, such as *Template* (using drag and drop functionality) and click **Save** when you complete the form. The workflow now runs and refreshes the fields with data from the workflow. In this way fields are chained together. When you *order* a service catalog, the workflow runs in the background to refresh the template.
+
+### Creating Service Catalog Items using an Embedded Workflow
+
+You can create a generic service catalog item that uses a created dynamic dialog and an embedded workflow. To use previously set up workflows and to view a service catalog item, you need to *order* the item. Complete the following steps to order a service catalog item that uses embedded workflows:
```

Comment:
> that uses a created dynamic dialog 

It isn't possible to use a dynamic dialog that isn't created, so I think the `uses a created dynamic dialog` is redundant

Reply:
I have removed this text. The sentence is now: 

> You can create a generic service catalog item that uses an embedded workflow.